html/get_alerts.php

```diff
@@ -126,19 +132,26 @@ function submitMe(dbid, handled)
 	</table>
 <?php
 	
-    echo ('<table class=TFtable width=100% halign=left>');
+    echo ('<table class=TFtable width=100% halign=left id=table_detail>');
         echo ('<tr>');
 			echo ('<td nowrap><b>First seen</b></td><td nowrap><b>Last seen</b></td><td nowrap><b>Host</b></td><td nowrap><b>Threat</b></td><td nowrap><b>Indicators</b></td><td nowrap><b>Is reviewed?</b></td>');
 		echo ('</tr>');
 		
     if ($obj[0] != 'none'){
-        foreach ($obj as $alert){  
+		$i = 0;
+        foreach ($obj as $alert){
+				$i++;
 				if ($alert[9] == "0"){
 					$checkbox = "<input type=checkbox name=handled id=".$alert[0]." value=0 onclick='submitMe(".$alert[0].", 1);'>";
+					$rev = "No";
 				}else{
 					$checkbox = "<input type=checkbox name=handled id=".$alert[0]." value=1 onclick='submitMe(".$alert[0].", 0);' checked>";
+					$rev = "Yes";
 				}
-            echo ('<tr><td nowrap>'.date('Y/m/d H:i:s', $alert[2]).'</td>'.'<td nowrap>'.date('Y/m/d H:i:s', $alert[3]).'</td>'.'<td nowrap>'.$alert[7].'</td>'.'<td nowrap>'.$alert[6].'</td>'.'<td><div class="text-content short-text">'.str_replace('|','| ',$alert[8]).'</div><div class="show-more"><a href="#">Show more</a></div></td><td nowrap>'.$checkbox.'</td></tr>');
+            echo ('<tr style="cursor: pointer;" onclick=show_hide_row("hidden_row'.$i.'"); title="Click for more details"><td nowrap>'.date('Y/m/d H:i:s', $alert[2]).'</td>'.'<td nowrap>'.date('Y/m/d H:i:s', $alert[3]).'</td>'.'<td nowrap>'.$alert[7].'</td>'.'<td nowrap>'.$alert[6].'</td>'.'<td><div class="text-content short-text">'.str_replace('|','| ',$alert[8]).'</div><div class="show-more"><a href="#">Show more</a></div></td><td nowrap>'.$checkbox.'</td></tr>');
```

Comment:
I added a small css style tag for the users to know that they can click on the rows of the table.

`style="cursor: pointer;"`